src/AssetsTransferApi.ts

```diff
@@ -408,7 +458,7 @@ export class AssetsTransferApi {
 		/**
 		 * Check if the origin is a parachain, and the destination is a system parachain.
 		 */
-		if (_api.query.polkadotXcm && isDestIdSystemPara) {
+		if (_api.query.polkadotXcm || (_api.query.xTokens && isDestIdSystemPara)) {
```

Comment:
I think this needs to be `(_api.query.polkadotXcm || _api.query.xTokens) && isDestIdSystemPara)`.